### DISCUSSION TITLE
Welcome to Github Discussions on Ledger Live Mobile

### DISCUSSION THREAD

**Discussion by cthepot** [LedgerHQ/ledger-live-mobile#1523] [General]
## :rocket: Welcome to Github Discussions on Ledger Live Mobile.

Feel free to share your thoughts and feedback on Ledger Live Mobile.

Discussions is a place for:
- [x] Feature requests
- [x] UX/UI suggestions
- [x] Questions about features

For :spider::bug: bugs, feel free to report them directly in the issues tab.

**Comment by SLVR1984** [comment]
Hello!I would really appreciate some expert advice from any of you concerning my problem with lost BTC.I apologyse for my simple explanation about the whole situation as my knowledge of the IT language and know how is almost non existent.I'd call it basic knowledge but probably not even that.In short,i am able to buy and exchange,send/receive and that's about it.But long story short-i had about 1000(got down to 700 but it was back in April so now it should be worth a bit more)usd worth of BTC on my mobile app.I had funds in Okex and transfered them to the mobile app in multiple transactions without being in sync with the desktop version(thinking back,that was a mistake to get them to ledger without beiing in sync with the desktop.But u can receive coins without validating with my Ledger Nano S.Everything was fine but i tried all kinds of ways to connect my device with desktop and also mobile app and never saw the amount of actual BTC on the desktop app,said it was all in sync but desktop showed a balance of O.And one day i got frustrated because i needed some money and wanted to transact some out of my mobile wallet,can't do that without validating with Nano S,angers me that i can easly put money in but transfering out is not possible.So i saw in settings that ur funds wont dissapear whatever u do:Reistall the app and so on,the funds are on the chain.So i deleted my mobile app and reinstalled.Balance-0.Also,would be easy to prove if i took even one photo of the funds on mobile but i took none,because i've read that Ledger is one of the best and i was under the impression that i would still have those funds in my wallet.Fortunately what i do have at home on my desktop is  a whole folder full of allkinds of files which came from saving history etc.All the people here would know where and what to look for by reading the code but i'm clueless.And also,is it possible that when i deleted then the coins went down the drain or are they still somewhere?And also,could someone please tell me what should i do next or should i forget about those coins once and for all.Contacting support again is pointless because i don't know how i would get all this data they wanted to see obviously.Thank you if u were able to make some sense of this story and i'm open to all suggestions✌️

**Reply by johdah** [reply]
I don't think this is the right place to ask for help but to try to be helpful. The coins are never stored on your device, it just holds your key (24 words). So if you have sent the coins to that as long as you have the 24 words you can never lose it (if you don't give those words to someone else). 

What I would do if I were you is to go to Okex and find the withdrawal history (have never used Okex so they would need to guide you). There you should be able to see a transaction ID and see the address you sent to. Make sure that you sent it to the right address (the same as in Ledger Live) otherwise there is nothing you can do. 

If they came to the right address they really should appear in Ledger Live and if not Ledger support should be the place to ask. If Ledger ask for information about the transaction you should probably be able to get that from Okex. Or at least Ledger should be able to guide you if it's related to Ledger Live.

Good luck!

**Reply by SLVR1984** [reply]
Yeah,i can see it that Github is site where actual coding bugs are discussed/solved by smart people behind the scenes so that it would be as easy as possible for dummies like me to also be able to operate these apps.But i knew that i'd get the best responce here because i didn't want to contact support again,they have been helpful and no complaints,the fact is that they just can't give such simple answers like u can because they have to be really careful with their advice and politically correct.Which i totally understand.But thanks a lot for the reply,and as i said,simple and informative👍I sort of knew all that but i'd reached a point where i needed a step by step guide to recover the coins(upset at myself a bit and that never solves anything).I have all the codes and 24 word key so i'm confident now that i'll get if fixed.Also i did some reading about SegWit and Native SegWit,i need to make that transaction in the app as well when i locate them.At least i'm learning something from these mistakes,aga hopefully one day i can post something that actually belongs here😃.Thanks againg and take care!✌️
